### PR TITLE
Added TC-BitRixTaskTestOC

### PR DESCRIPTION
**Pull request by Copeichina**
Created a TC for verifying that we can check in Task Page "High Priority" check box

### CODE REVIEW

**Comment by ivjava** [comment]
Check naming convention

File: src/test/java/com/bitrix/tests/BitRixTaskPageTestOC.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+public class BitRixTaskPageTestOC extends TestBase {
+    @Test
+    public void test() {
+        loginPage.login("hr_user");
+
+        WebElement task = driver.findElement(By.xpath("//span[.='Task']"));
+        task.click();
+
+        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("(//input[@type='checkbox'])[13]")));
```
Comment (ivjava) [inline src/test/java/com/bitrix/tests/BitRixTaskPageTestOC.java:19]: No locators in test - follow P.O.M

File: src/test/java/com/bitrix/tests/BitRixTaskPageTestOC.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class BitRixTaskPageTestOC extends TestBase {
+    @Test
+    public void test() {
+        loginPage.login("hr_user");
+
+        WebElement task = driver.findElement(By.xpath("//span[.='Task']"));
```
Comment (ivjava) [inline src/test/java/com/bitrix/tests/BitRixTaskPageTestOC.java:16]: I created PortalPage for this part - will/can delete later or update project - did you pull before pushing?

**Review by ivjava — CHANGES_REQUESTED** [review]
Please follow page object model. 
Ignore the naming convention comment if you got it - I don't know :)

**Comment by Copeichina** [comment]
> Check naming convention

Thank you I will check the naming convention

**Comment by Copeichina** [comment]
Was taken in consideration.

**Pull request closed without merging**